Makefile

```diff
@@ -16,6 +16,7 @@ EFFECTIVE_VERSION           := $(VERSION)-$(shell git rev-parse HEAD)
 LD_FLAGS                    := "-w $(shell bash $(GARDENER_HACK_DIR)/get-build-ld-flags.sh "" $(REPO_ROOT)/VERSION "$(EXTENSION_PREFIX)")"
 LEADER_ELECTION             := false
 IGNORE_OPERATION_ANNOTATION := true
+ARCH                		?= amd64
```

Comment:
Should we add the OS to the variable?
`PLATFORM ?= linux/amd64`

Reply:
as discussed we'll keep linux hard-coded for now.